micromamba/etc/profile.d/mamba.sh.in

```diff
@@ -1,4 +1,10 @@
-export MAMBA_ROOT_PREFIX="@CMAKE_INSTALL_PREFIX@"
+if [ -z "${MAMBA_ROOT_PREFIX}" ]; then
+    echo "WARNING: MAMBA_ROOT_PREFIX is not set."
+    echo "WARNING: Please set `MAMBA_ROOT_PREFIX` to the root of your installation."
+    echo "WARNING: For now continuing with `MAMBA_ROOT_PREFIX` set to `@CMAKE_INSTALL_PREFIX@`."
+    export MAMBA_ROOT_PREFIX="@CMAKE_INSTALL_PREFIX@"
+fi
```

Comment:
What do people think of this?

Reply:
I don't have any strong opinion about this. We can still revert it later if people are annoyed by the warning.